courses/admin.py

```diff
@@ -192,6 +192,18 @@ def has_delete_permission(self, request, obj=None):
         return False
 
 
+class CourseRunEnrollmentAuditInline(admin.TabularInline):
+    """Inline editor for CourseRunEnrollmentAudit"""
+
+    model = CourseRunEnrollmentAudit
+    readonly_fields = [
+        "enrollment_id",
+        "created_on",
+    ]
+    min_num = 0
+    extra = 0
```

Comment:
Can we add also add code to prevent add or delete? This table is for audit, so I don't think it should be editable for admin unless there is a use case?

```
    can_delete = False
    can_add = False

    def has_add_permission(self, request, obj=None):
        return False
```